src/main/scala/com/gu/commercial/branding/BrandingFinder.scala

```diff
@@ -63,17 +63,21 @@ object SponsorshipHelper {
     dateLaterThanThreshold getOrElse true
   }
 
-  def findSectionSponsorship(edition: String, publishedDate: Option[CapiDateTime])
+  def findSponsorshipFromSection(edition: String, publishedDate: Option[CapiDateTime])
     (section: Section): Option[Sponsorship] = {
     section.activeSponsorships.flatMap(_.find { s =>
       isTargetingEdition(edition)(s) && isTargetingDate(publishedDate)(s)
     })
   }
 
-  def findTagsSponsorship(tags: Seq[Tag], publishedDate: Option[CapiDateTime], edition: String): Option[Sponsorship] =
-    tags.toStream.flatMap(t => findTagSponsorship(edition, publishedDate)(t)).headOption
+  def findSponsorshipFromTags(
+    tags: Seq[Tag],
+    publishedDate: Option[CapiDateTime],
+    edition: String
+  ): Option[Sponsorship] =
+    tags.toStream.flatMap(t => findSponsorshipFromTag(edition, publishedDate)(t)).headOption
```

Comment:
Could write `tags.toStream.flatMap(findSponsorshipFromTag(edition, publishedDate)(_)).headOption` which may or may not be clearer depending on your point of view.

Out of interest, what is the difference between `.toStream` and `.view` here?